resources/changelog.html

```diff
@@ -1,821 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-<meta charset="utf-8">
-<style type="text/css">
-	body {color: #9d9995;}
-	a {color: #268bd2;}
-	a:visited { color: #1E6A9E; }
-	h1 { font-size:  xx-large; color: #268bd2; margin-bottom:1em; }
-	h2 { font-size: large; color: #268bd2; margin-top:1em; margin-bottom:0em; }
-	p { margin-bottom: 0.2em; margin-top: 0.1em; }
-	div { margin: 0; padding: 0; }
-	pre { margin-top: 0.3em; margin-bottom: 0.3em; }
-	code { color: #E0E0DB; background-color: #4a4a4f; padding-left: 0.4em; padding-right: 0.4em; }
-	div.highlight { background-color: #4a4a4f; padding-top: 0.1em; padding-bottom: 0.1em; }
-	.hll { background-color: #49483e }
-	.c { color: #75715e } /* Comment */
-	.err { color: #960050; background-color: #1e0010 } /* Error */
-	.k { color: #66d9ef } /* Keyword */
-	.l { color: #ae81ff } /* Literal */
-	.n { color: #f8f8f2 } /* Name */
-	.o { color: #f92672 } /* Operator */
-	.p { color: #f8f8f2 } /* Punctuation */
-	.ch { color: #75715e } /* Comment.Hashbang */
-	.cm { color: #75715e } /* Comment.Multiline */
-	.cp { color: #E22453; font-weight: bold } /* Comment.Preproc */
-	.cpf { color: #E6DB74; font-style: italic } /* Comment.PreprocFile */
-	.c1 { color: #75715e } /* Comment.Single */
-	.cs { color: #75715e } /* Comment.Special */
-	.gd { color: #f92672 } /* Generic.Deleted */
-	.ge { font-style: italic } /* Generic.Emph */
-	.gi { color: #a6e22e } /* Generic.Inserted */
-	.gs { font-weight: bold } /* Generic.Strong */
-	.gu { color: #75715e } /* Generic.Subheading */
-	.kc { color: #66d9ef } /* Keyword.Constant */
-	.kd { color: #66d9ef } /* Keyword.Declaration */
-	.kn { color: #f92672 } /* Keyword.Namespace */
-	.kp { color: #66d9ef } /* Keyword.Pseudo */
-	.kr { color: #66d9ef } /* Keyword.Reserved */
-	.kt { color: #66d9ef } /* Keyword.Type */
-	.ld { color: #e6db74 } /* Literal.Date */
-	.m { color: #ae81ff } /* Literal.Number */
-	.s { color: #e6db74 } /* Literal.String */
-	.na { color: #a6e22e } /* Name.Attribute */
-	.nb { color: #f8f8f2 } /* Name.Builtin */
-	.nc { color: #a6e22e } /* Name.Class */
-	.no { color: #66d9ef } /* Name.Constant */
-	.nd { color: #a6e22e } /* Name.Decorator */
-	.ni { color: #f8f8f2 } /* Name.Entity */
-	.ne { color: #a6e22e } /* Name.Exception */
-	.nf { color: #a6e22e } /* Name.Function */
-	.nl { color: #f8f8f2 } /* Name.Label */
-	.nn { color: #f8f8f2 } /* Name.Namespace */
-	.nx { color: #a6e22e } /* Name.Other */
-	.py { color: #f8f8f2 } /* Name.Property */
-	.nt { color: #f92672 } /* Name.Tag */
-	.nv { color: #f8f8f2 } /* Name.Variable */
-	.ow { color: #f92672 } /* Operator.Word */
-	.w { color: #f8f8f2 } /* Text.Whitespace */
```

Comment:
Why did you delete the changelog?

Reply:
It is always marked as changed on my computers, guess I staged it by mistake

Reply:
That is most likely causes by an older version of either markdown or gfm python modules.